### ISSUE TITLE
apps:clone fails when trying to copy the cache dir

### ISSUE THREAD

**Issue by elia** [dokku/dokku#2711]
**Description of problem:**

`dokku apps:clone` fails when trying to copy the cache dir

output with trace on:

```
⤑ bin/dokku apps:clone master master-copy
Creating master-copy... done
✂️
+ local phase_file_path=/home/dokku/master-copy2/DOCKER_OPTIONS_DEPLOY
+ [[ -f /home/dokku/master-copy2/DOCKER_OPTIONS_DEPLOY ]]
+ touch /home/dokku/master-copy2/DOCKER_OPTIONS_DEPLOY
+ echo --restart=on-failure:10
+ local all_phase_options=--restart=on-failure:10
+ echo -e --restart=on-failure:10
+ sed '/^$/d'
+ sort -u
+ cp -a /home/dokku/master/. /home/dokku/master-copy2
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/cache': Permission denied
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/specifications': Permission denied
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/bin': Permission denied
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/extensions': Permission denied
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/build_info': Permission denied
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/doc': Permission denied
cp: cannot access '/home/dokku/master/./cache/cedar-14/vendor/bundle/ruby/2.3.0/gems': Permission denied
Connection to XXXXXXXXX closed.
```

Let me add the app is ruby on rails and having it clone the cache dir would save significant time avoiding reinstalling all dependencies.

---

**dokku report**

```
~# dokku report
-----> uname: Linux XXXXXHOSTNAMEXXXX 3.13.0-57-generic #95-Ubuntu SMP Fri Jun 19 09:28:15 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
-----> memory: 
             total       used       free     shared    buffers     cached
Mem:         16048      14698       1350        676        442       3394
-/+ buffers/cache:      10861       5187
Swap:            0          0          0
-----> docker version: 
Client:
 Version:      1.12.3
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   6b644ec
 Built:        Wed Oct 26 21:44:32 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.3
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   6b644ec
 Built:        Wed Oct 26 21:44:32 2016
 OS/Arch:      linux/amd64
-----> docker daemon info: 
Containers: 91
 Running: 64
 Paused: 0
 Stopped: 27
Images: 485
Server Version: 1.12.3
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 760
 Dirperm1 Supported: false
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: overlay null bridge host
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: apparmor
Kernel Version: 3.13.0-57-generic
Operating System: Ubuntu 14.04.3 LTS
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 15.67 GiB
Name: XXXXXHOSTNAMEXXXX
ID: NWZB:SP2Z:QDH2:PE4Z:F4VH:NWSJ:EPMN:MFRU:WMXQ:DDAU:DZ5V:WQ2Y
Docker Root Dir: /var/lib/docker
Debug Mode (client): true
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Insecure Registries:
 127.0.0.0/8
-----> sigil version: 0.4.0
-----> herokuish version: 
herokuish: 0.3.27
buildpacks:
  heroku-buildpack-multi     v1.0.0
  heroku-buildpack-ruby      v155
  heroku-buildpack-nodejs    v98
  heroku-buildpack-clojure   v76
  heroku-buildpack-python    v99
  heroku-buildpack-java      v51
  heroku-buildpack-gradle    v21
  heroku-buildpack-grails    v21
  heroku-buildpack-scala     v76
  heroku-buildpack-play      v26
  heroku-buildpack-php       v120
  heroku-buildpack-go        v62
  heroku-buildpack-erlang    fa17af9
  buildpack-nginx            v8
-----> dokku version: 0.9.2
-----> dokku plugins: 
 !     Deprecated: Please use plugin:list
plugn: 0.3.0
  00_dokku-standard    0.9.2 enabled    dokku core standard plugin
  20_events            0.9.2 enabled    dokku core events logging plugin
  apps                 0.9.2 enabled    dokku core apps plugin
  build-env            0.9.2 enabled    dokku core build-env plugin
  certs                0.9.2 enabled    dokku core certificate management plugin
  checks               0.9.2 enabled    dokku core checks plugin
  common               0.9.2 enabled    dokku core common plugin
  config               0.9.2 enabled    dokku core config plugin
  docker-options       0.9.2 enabled    dokku core docker-options plugin
  dokku-git-rev        1.0.0 enabled    adds the current git hash as an environment variable
  domains              0.9.2 enabled    dokku core domains plugin
  enter                0.9.2 enabled    dokku core enter plugin
  git                  0.9.2 enabled    dokku core git plugin
  logs                 0.9.2 enabled    dokku core logs plugin
  memcached            1.0.0 enabled    dokku memcached service plugin
  mysql                1.0.0 enabled    dokku mysql service plugin
  named-containers     0.9.2 enabled    dokku core named containers plugin
  nginx-vhosts         0.9.2 enabled    dokku core nginx-vhosts plugin
  plugin               0.9.2 enabled    dokku core plugin plugin
  postgres             1.0.0 enabled    dokku postgres service plugin
  proxy                0.9.2 enabled    dokku core proxy plugin
  ps                   0.9.2 enabled    dokku core ps plugin
  redis                1.0.0 enabled    dokku redis service plugin
  repo                 0.9.2 enabled    dokku core repo plugin
  shell                0.9.2 enabled    dokku core shell plugin
  ssh-keys             0.9.2 enabled    dokku core ssh-keys plugin
  storage              0.9.2 enabled    dokku core storage plugin
  tags                 0.9.2 enabled    dokku core tags plugin
  tar                  0.9.2 enabled    dokku core tar plugin
```

Environment details (AWS, VirtualBox, physical, etc.): Digital Ocean

How was Dokku installed?: rpm

How reproducible: consistently

Steps to Reproduce: 
1. `dokku apps:clone` on a rails app

**Comment by josegonzalez** [comment]
The cache dir *cannot* be copied due to the aformentioned ownership issues, and I am loathe to adding another `sudoers` entry to copy cache.

The easiest fix is to just avoid copying the cache directory for the clone command for now.

**Comment by josegonzalez** [comment]
Closed by #2730.
